### ISSUE TITLE
bug: building using flatpak-builder

### ISSUE THREAD

**Issue by libreajans** [GradienceTeam/Gradience#312]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I build Gradience from source with flatpak-builder. But there is an error.

```
flatpak run com.github.GradienceTeam.Gradience.Devel 
Traceback (most recent call last):
  File "/app/bin/gradience", line 64, in <module>
    from gradience import main
  File "/app/lib/python3.9/site-packages/gradience/main.py", line 35, in <module>
    from .welcome import GradienceWelcomeWindow
  File "/app/lib/python3.9/site-packages/gradience/welcome.py", line 21, in <module>
    from gi.repository import Gtk, Adw, Gio, Gdk, Glib
  File "/usr/lib/python3.9/site-packages/gi/importer.py", line 136, in load_module
    raise ImportError('cannot import name %s, '
ImportError: cannot import name Glib, introspection typelib not found
```

### To Reproduce

1. Open Terminal
2. Run `git clone https://github.com/GradienceTeam/Gradience.git && cd Gradience`
3. Install the `42` version of GNOME SDK: `flatpak install org.gnome.Sdk/x86_64/42 org.gnome.Platform/x86_64/42`
4. Run: `flatpak-builder --install --user --force-clean repo/ build-aux/flatpak/com.github.GradienceTeam.Gradience.Devel.json`

### Screenshots

![Ekran Görüntüsü - 2022-08-31 23-28-36](https://user-images.githubusercontent.com/6218679/187775896-4f6d15c1-9f0e-403c-a4bb-ef52b5beea40.png)

### OS

Ubuntu 22.04.1

### Version

from source

### Installation method

Flatpak from manual build - main

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tfuxu** [comment]
@libreajans 
Try installing `gobject-introspection`. On Ubuntu it should just be: `sudo apt install gobject-introspection`

**Comment by tfuxu** [comment]
Looks like this issue got fixed in commit 295a4b5165732073fdddd037f123a29c2f2d3aa2.

**Comment by 0xMRTT** [comment]
Yes, 😅 sorry for this guys 😶
